### ISSUE TITLE
[MAS2.1.1] 'Encrypt your keys' check-box is not accessible using keyboard.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#734]
**Actual Result:**
'Encrypt your keys' check-box present in 'New Bot Configuration' dialog-box is not accessible using keyboard.

**Expected Result:**
'Encrypt your keys' check-box present in 'New Bot Configuration' dialog-box should be accessible using keyboard. 

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome screen will open up.
3. Navigate to 'Create a new bot configuration' link and select it.
4. In 'New Bot Configuration' dialog-box, navigate using Tab key and notice that focus does not come at 'Encrypt your keys' check-box. 

**User Impact:**
Keyboard dependent users won't be able to use the controls which are not accessible using keyboard in any way.

**Test Environment:**
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge

**Impacted MAS Link:**
[MAS 2.1.1 – Keyboard (36)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

**Attachments:**
[MAS2.1.1_BotFrameworkEmulator_NewBotConfiguration_EncryptYourKeys_CheckBox.zip](https://github.com/Microsoft/BotFramework-Emulator/files/2266011/MAS2.1.1_BotFrameworkEmulator_NewBotConfiguration_EncryptYourKeys_CheckBox.zip)

**Comment by corinagum** [comment]
Fixed
